src/sentry/runner/commands/run.py

```diff
@@ -253,6 +253,61 @@ def taskworker(rpc_host: str, max_task_count: int, **options: Any) -> None:
         raise SystemExit(exitcode)
 
 
+@run.command()
+@log_options()
+@configuration
+@click.option(
+    "--repeat",
+    type=int,
+    help="Number of messages to send to the kafka topic",
+    default=100,
```

Comment:
```suggestion
    default=1,
```

I think we should have the default start off small. I think it is more likely that someone will want to do local testing one task at a time instead of getting 100.

Reply:
Makes sense, updated 👍